docs/HOWTO.md

```diff
@@ -23,19 +23,62 @@ The environment should start up with instructions on what to do next.
 
 # Kernels
 
+After initializing your project with a flake template, it should have a `kernels` directory which contains all the kernels.
```

Comment:
This is the bit that confused me initially in #332, because currently after nix init flake there are no derivations in the kernels dir except for python.

So instead I'd suggest here saying something like

```suggestion
After initializing your project with a flake template, it should have a `kernels` directory, which contains `python.nix` kernel. You can find more kernel derivations in `kernels` directory at the root of this repository.
```